### DISCUSSION TITLE
Σχετικα με την τριτη ενοτητα "Υπενθυμιση" στο template των Issues.

### DISCUSSION THREAD

**Discussion by illidan-git99** [ioniodi/sitegr#383] [Q&A] [answered]
Στις υπενθυμισεις γραφει οτι "εχω ελεγξει οτι το περιεχομενο των αλλαγων μου δεν το εχει δηλωσει καποιος αλλος σε προηγουμενο ανοιχτο ή κλειστο issue", αλλα ειδα [ενα](https://github.com/ioniodi/sitegr/issues/376#issuecomment-1090031647) σχολιο του κ. @epidrome που προτεινε να παρουμε παλια θέματα που ειτε δεν εγιναν ειτε εχουν bugs και να εργαστουμε πανω σε αυτα. Αυτο με εκανε να αναρωτηθω μηπως θα επρεπε, τοτε, να αλλαξει το κειμενο αυτης της υπενθυμισης για να κατοπτριζει τη πραγματικη απαιτηση; 

![image](https://user-images.githubusercontent.com/72458452/162332175-3958cdcf-3ddd-4310-b705-6f9f20f9c693.png)

Και να γινοταν κατι σαν:
- Πρεπει να ελεγχουμε αν το issue που προτεινουμε δεν εχει ηδη υλοποιηθει σε προηγουμενο ανοιχτο ή κλειστο issue **και** αν εχει υλοποιηθει, δεν εχει bug **και** αν εχει υλοποιηθει, δεν μπορει να βελτιωθει περαιτερω. 

Ειναι λιγο μακριναρι αυτο που προτεινω, αλλα μπορει να δουλεψει ως sample μεχρι να βρεθει κατι πιο συνοπτικο. Γραφει ακριβως καποια απο (δεν ξερω αν υπαρχουν άλλα) τα απαιτουμενα και καλο θα ηταν οι υπενθυμισεις να ειναι σαφεις.

**Comment by diogenisAl** [comment, marked as answer]
Η αρχική ιδέα του:
> εχω ελεγξει οτι το περιεχομενο των αλλαγων μου δεν το εχει δηλωσει καποιος αλλος σε προηγουμενο ανοιχτο ή κλειστο issue

ήταν να μην προτείνουν το ίδιο issue 2 ή περισσότεροι φοιτητές στο ίδιο παραδοτέο του τρέχοντος εξαμήνου. Ωστόσο, αν κάτι προτάθηκε στο παρελθόν και δεν ολοκληρώθηκε, τότε είναι διαθέσιμο.

Σε ευχαριστούμε @p19kala για την επισήμανση.
